README.md

```diff
@@ -85,8 +75,40 @@ grunt ci             # run the service tests and polyfills tests on a large set
 
 We run the tests [on CircleCI][ci].  `grunt ci` must pass before we merge a pull request.
 
-Release Process
----------------
+
+## Real User Monitoring
+
+We have shipped experimental support for using RUM to monitor feature support and performance in browsers.  This involves a number of parts, all activated by the presence `RUM_MYSQL_DSN` env var:
+
+* **RUM client code**: a [small snippet](lib/rumTemplate.js.handlebars) of legacy-compatible code that will evaluate feature detects on the client, sample resource timing data, and beacon the results back to the service. This is shipped as part of the Node app.
+* **Beacon endpoint**: an [endpoint to collect RUM data](fastly-config.vcl), terminated at the CDN, logging query data out to Amazon S3 to avoid overloading the backend.  To clarify, the backend node server does not provide a route handler for the RUM data collection URL, so this is shipped when we deploy VCL to Fastly.  It also requires log streaming to be configured in the Fastly UI.
```

Comment:
Could we add `#L22` to the link, that way we direct the reader to the part of the file which is doing this work.